zh-CN/mvc/view/static.md

```diff
@@ -7,17 +7,17 @@ sort: 3
 
 Go 语言内部其实已经提供了 `http.ServeFile`，通过这个函数可以实现静态文件的服务。beego 针对这个功能进行了一层封装，通过下面的方式进行静态文件注册：
 
-	beego.SetStaticPath("/static","public")
+	beego.SetStaticPath("public","/static")
```

Comment:
while I tested in version 1.6
the `/` for first param is optional and `/` for second param is not acceptable